include/Core/Controller.php

```diff
@@ -164,6 +165,31 @@ public function load_taxonomy_term() {
 		die;
 	}
 
+	/**
+	 * Ajax call back function for getting taxonomies meta to load select2 options.
+	 *
+	 * @since 6.0.1
+	 */
+	public function load_taxonomy_term_meta() {
+		$response = array();
+
+		$term_id = sanitize_text_field( $_GET['term_id'] );
```

Comment:
Use `absint` function instead of `sanitize_text_field`

Reply:
Fixed